src/Octoshift/Services/GithubApi.cs

```diff
@@ -1039,6 +1039,39 @@ public virtual async Task<IEnumerable<CodeScanningAlertInstance>> GetCodeScannin
             .ToListAsync();
     }
 
+    public virtual async Task<IEnumerable<DependabotAlert>> GetDependabotAlertsForRepository(string org, string repo)
+    {
+        var url = $"{_apiUrl}/repos/{org.EscapeDataString()}/{repo.EscapeDataString()}/dependabot/alerts?per_page=100";
+        return await _client.GetAllAsync(url)
+            .Select(BuildDependabotAlert)
+            .ToListAsync();
+    }
+
+    public virtual async Task UpdateDependabotAlert(string org, string repo, int alertNumber, string state, string dismissedReason = null, string dismissedComment = null)
+    {
+        if (!DependabotAlertState.IsOpenOrDismissed(state))
+        {
+            throw new ArgumentException($"Invalid value for {nameof(state)}");
+        }
+
+        if (DependabotAlertState.IsDismissed(state) && !DependabotAlertState.IsValidDismissedReason(dismissedReason))
+        {
+            throw new ArgumentException($"Invalid value for {nameof(dismissedReason)}");
+        }
+
+        var url = $"{_apiUrl}/repos/{org.EscapeDataString()}/{repo.EscapeDataString()}/dependabot/alerts/{alertNumber}";
+
+        var payload = state == "open"
+            ? (new { state })
+            : (object)(new
```

Comment:
Do we need to explicitly cast to an object here? What would happen if we didn't?